src/main/scala/Timestamp.scala

```diff
@@ -0,0 +1,14 @@
+package chess
+
+case class Timestamp(value: Long) extends AnyVal with Ordered[Timestamp] {
+  // Convert to centi duration with simple int round.
+  def to(end: Timestamp) =
+    if (end.value > value) Centis((end.value - value + 5) / 10)
```

Comment:
this is probably overkill...  should we bother to round millis or just truncate?